Formula/pcp.rb

```diff
@@ -0,0 +1,54 @@
+class Pcp < Formula
+  desc "Performance Co-Pilot is a system performance and analysis framework"
+  homepage "https://pcp.io/"
+  url "https://dl.bintray.com/pcp/source/pcp-4.1.3.src.tar.gz"
+  sha256 "ca593d65ccaf6f481d55720019e317679afeadd589aa37c878ec1e8629a1e75c"
+
+  depends_on "pkg-config" => :build
+  depends_on "nss"
+  depends_on "python"
+  depends_on "qt"
+  depends_on "readline"
+  depends_on "xz"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "CC=clang", "CXX=clang", "--without-manager"
```

Comment:
Doesn't `configure` pick up the `CC` and `CXX` values from environment?
(And if it doesn't: shouldn't `CXX` be `clang++`?)

Reply:
We were using this for initial testing but the formula works without it, updated in the latest commit.